### ISSUE TITLE
Exception on kitchen create: Windows Server 2012 R2 box

### ISSUE THREAD

**Issue by djenriquez** [test-kitchen/test-kitchen#696]
Seup is identical to [#695](https://github.com/test-kitchen/test-kitchen/issues/695) with the exception of ```vm_hostname: false``` being commented out. Also, when ```vm_hostname: false``` is left out, ```==> default: Setting hostname...``` takes quite a bit of time to complete.

```
-----> Starting Kitchen (v1.4.0.rc.1)
-----> Creating <default-windows-2012R2>...
       Bringing machine 'default' up with 'virtualbox' provider...
       ==> default: Importing base box 'WindowsServ2012R2'...
==> default: Matching MAC address for NAT networking...
       ==> default: Setting the name of the VM: kitchen-djiis-default-windows-2012R2_default_1429258197619_1595
       ==> default: Clearing any previously set network interfaces...
       ==> default: Preparing network interfaces based on configuration...
           default: Adapter 1: nat
           default: Adapter 2: hostonly
       ==> default: Forwarding ports...
           default: 3389 => 3389 (adapter 1)
           default: 22 => 2222 (adapter 1)
           default: 5985 => 5985 (adapter 1)
           default: 5985 => 55985 (adapter 1)
       ==> default: Running 'pre-boot' VM customizations...
       ==> default: Booting VM...
       ==> default: Waiting for machine to boot. This may take a few minutes...
       ==> default: Machine booted and ready!
       ==> default: Checking for guest additions in VM...
       ==> default: Setting hostname...
       ==> default: Configuring and enabling network interfaces...
       Guest-specific operations were attempted on a machine that is not
       ready for guest communication. This should not happen and a bug
       should be reported.
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'WindowsServ2012R2'...
==> default: Matching MAC address for NAT networking...
==> default: Setting the name of the VM: kitchen-djiis-default-windows-2012R2_default_1429258197619_1595
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 3389 => 3389 (adapter 1)
    default: 22 => 2222 (adapter 1)
    default: 5985 => 5985 (adapter 1)
    default: 5985 => 55985 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
STDERR: Guest-specific operations were attempted on a machine that is not
ready for guest communication. This should not happen and a bug
should be reported.
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1]
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration
```

**Comment by tyler-ball** [comment]
Hey @djenriquez - like #695 can you try this again after installing the latest ChefDK?

**Comment by djenriquez** [comment]
@tyler-ball I believe the fix for #695 fixed this one as well. Thanks again. Closing.
